Gopkg.toml

```diff
@@ -20,7 +22,10 @@ required = [
   "k8s.io/code-generator/cmd/set-gen",
 ]
 
-ignored = ["github.com/kubernetes/repo-infra/kazel"]
+ignored = [
+  "github.com/kubernetes/repo-infra/kazel",
+  "github.com/google/protobuf/*",
```

Comment:
Is there a transitive dependency requiring `dep` to ignore?

Reply:
Hmm. I thought I had removed this, but I'll make sure it's actually gone. Perhaps it slipped into another commit as I was rebasing.